src/main/scala/Configs.scala

```diff
@@ -153,6 +153,7 @@ class DefaultConfig extends Config (
       }
       case BuildRoCC => Nil
       case RoccNMemChannels => site(BuildRoCC).map(_.nMemChannels).foldLeft(0)(_ + _)
+      case RoccNPTWPorts => site(BuildRoCC).map(_.nPTWPorts).foldLeft(0)(_ + _)
```

Comment:
I was trying to be consistent with other parameters around: RoccNMemChannels, RoccNCSRs.